ovh/data_ovhcloud_connect_config_pop.go

```diff
@@ -0,0 +1,92 @@
+package ovh
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+
+	"github.com/hashicorp/terraform-plugin-framework/datasource"
+	ovhtypes "github.com/ovh/terraform-provider-ovh/v2/ovh/types"
+)
+
+var _ datasource.DataSourceWithConfigure = (*ovhcloudConnectConfigPopDataSource)(nil)
+
+func NewOvhcloudConnectConfigPopDataSource() datasource.DataSource {
+	return &ovhcloudConnectConfigPopDataSource{}
+}
+
+type ovhcloudConnectConfigPopDataSource struct {
+	config *Config
+}
+
+func (d *ovhcloudConnectConfigPopDataSource) Metadata(ctx context.Context, req datasource.MetadataRequest, resp *datasource.MetadataResponse) {
+	resp.TypeName = req.ProviderTypeName + "_ovhcloud_connect_config_pop"
```

Comment:
```suggestion
	resp.TypeName = req.ProviderTypeName + "_ovhcloud_connect_config_pops"
```

since you're retrieving all the pops

Reply:
done